pkg/oci/utils.go

```diff
@@ -431,17 +431,15 @@ func ContainerConfig(ocispec CompatOCISpec, bundlePath, cid, console string, det
 }
 
 // StatusToOCIState translates a virtcontainers container status into an OCI state.
-func StatusToOCIState(status vc.ContainerStatus) (spec.State, error) {
-	state := spec.State{
+func StatusToOCIState(status vc.ContainerStatus) spec.State {
+	return spec.State{
```

Comment:
I'm not even sure you need the return here do you? If you are filling out a return variable, doesn't it just 'fall out of the bottom' eu-natural? Having said that, leaving the return is fine - it makes it very clear what is going on :-)

Reply:
You can leave out the name of the variable on `return` if you've specified one in the signature but this function hasn't actually specified a name, just a type.